watchman/checks.py

```diff
@@ -9,6 +9,8 @@
 from django.core.mail import EmailMessage
 from django.db import connections
 
+import pika
```

Comment:
I would suggest the optional import. The check wouldn't be used anyways if this package or some other equivalent mechanism was available manage the connection.